README.md

```diff
@@ -484,6 +485,28 @@ $ python krun.py --dump-reboots examples/example_results.json.bz2
 8
 ```
 
+## Troubleshooting
+
+### `java.lang.UnsatisfiedLinkError` Error
+
+Finding the following strack trace in the logs during a benchmark is a sign that something has not been compiled with the ENABLE_JAVA option set to 1. 
```

Comment:
"strack" -> "stack" (please spell check your changes).

But I think we can make this sound a little more concise like this:
````
The following error in the log file is indicative that Krun has not been compiled with Krun support:

<the stack trace as before>

See Step 3 on how to build with Java support.
```